qiskit_optimization/problems/linear_expression.py

```diff
@@ -172,3 +185,25 @@ def evaluate_gradient(self, x: Union[ndarray, List, Dict[Union[int, str], float]
 
         # extract the coefficients as array and return it
         return self.to_array()
+
+    def bounds(self) -> ExpressionBounds:
```

Comment:
Would this be easier for callers as a property? Instead of doing `problem.objective.quadratic.bounds().upperbound` where `bounds()` needs to be done like that, it could instead be `problem.objective.quadratic.bounds.upperbound`

Reply:
Thanks. Property looks smart. I made it property.